CHANGELOG.txt

```diff
@@ -1,5 +1,16 @@
 List of significant changes and bug fixes going back to version 1.1.0
 
+Changes in version 3.2.1
+
+ This is a patch release. It fixes some bugs and has minor improvements. 
+
+    * #3686 Fixed Audacity crashing on startup with EXCEPTION_ACCESS_VIOLATION_READ on some systems.
```

Comment:
I would prefer simply `Fixed Audacity crashing on startup on some systems.`

The crash is not Windows specific, the EXCEPTION_ACCESS_VIOLATION_READ is.